scripts/index.js

```diff
@@ -100,15 +101,16 @@ function changeCategory() {
     
     let newDifficulty = document.getElementById("difficulty")
     let selectedDifficulty = newDifficulty[newDifficulty.selectedIndex].value;
-    
     let newCategory = document.getElementById("select-category");
     let selectedValue = newCategory[newCategory.selectedIndex].value;
-    generalURL = `https://my-little-cors-proxy.herokuapp.com/https://opentdb.com/api.php?amount=1&category=${selectedValue}&difficulty=${selectedDifficulty}&type=multiple`;
+    generalURL = `https://opentdb.com/api.php?amount=1&category=${selectedValue}&difficulty=${selectedDifficulty}&type=multiple`;
```

Comment:
revised generalURL, herokuapp proxy not needed
also added missing "=" to difficulty, was causing console to print error occasionally